client/layout/controller.js

```diff
@@ -245,7 +245,7 @@ export function updateLinkHref( item, nextQuery, excludedScreens ) {
 	if ( isWCAdmin ) {
 		const search = last( item.href.split( '?' ) );
 		const query = parse( search );
-		const path = query.path || 'dashboard';
+		const path = query.path || 'homepage';
```

Comment:
Should this check to see if the `homepage` feature is active first?

Reply:
Good call. I added a check in https://github.com/woocommerce/woocommerce-admin/pull/4288/commits/d67115a4ded108f8a574ba592437dacc13c74615